pkg/metrics/api_server_metrics.go

```diff
@@ -48,6 +48,24 @@ var KnownApiServerMetrics = map[string][]string{
 	"rest_client_request_status_codes":               {"code", "host", "method"},
 }
 
+var InterestingApiServerMetricsForE2E = []string{
```

Comment:
I guess that this [comment](https://github.com/kubernetes/kubernetes/pull/19261/files#r48809561) also applies here. Do you plan to change it?